pkg/snapshot/snapshot.go

```diff
@@ -8,11 +8,12 @@ import (
 	snapshotvolume "github.com/kubernetes-incubator/external-storage/snapshot/pkg/volume"
 	"github.com/libopenstorage/stork/drivers/volume"
 	"github.com/libopenstorage/stork/pkg/snapshot/controllers"
-	"github.com/portworx/sched-ops/k8s/errors"
+	schederrors "github.com/portworx/sched-ops/k8s/errors"
```

Comment:
Don't need to alias unless there is a conflict

Reply:
Update this but it's better to keep the same names over project.